perfkitbenchmarker/providers/yandexcloud/yc_disk.py

```diff
@@ -0,0 +1,122 @@
+# Copyright 2020 PerfKitBenchmarker Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Module containing classes related to YC disks.
+
+Disks can be created, deleted, attached to VMs, and detached from VMs.
+Use 'yc compute disk-types list' to determine valid disk types.
+"""
+
+import time
+import json
+import random
+import string
+
+from perfkitbenchmarker import disk
+from perfkitbenchmarker import flags
+from perfkitbenchmarker.providers import YC
+from perfkitbenchmarker.providers.yandexcloud import util
+
+FLAGS = flags.FLAGS
+
+NET_SSD = 'network-ssd'
+NET_HDD = 'network-hdd'
+
+DISK_TYPE = {disk.STANDARD: NET_HDD, disk.REMOTE_SSD: NET_SSD}
+
+DISK_METADATA = {
+    NET_HDD: {
+        disk.MEDIA: disk.HDD,
+        disk.REPLICATION: disk.ZONE,
+    },
+    NET_SSD: {
+        disk.MEDIA: disk.SSD,
+        disk.REPLICATION: disk.ZONE,
+    },
+}
+
+disk.RegisterDiskTypeMap(YC, DISK_TYPE)
+
+
+class YcDisk(disk.BaseDisk):
+  """Object representing an YC Disk."""
+
+  def __init__(self, disk_spec, name, zone, folder_id=None,
```

Comment:
Pytype, in this case

`(self, disk_spec: disk.BaseDiskSpec, name:str, zone:str, folder_id:str=None):`